solutions/CF Save the Nature.mdx

```diff
@@ -0,0 +1,214 @@
+---
+id: cf-1223C
+source: CF
+title: Save the Nature
+author: Jesse Choe, Kevin Sheng
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/66993)
+
+## Explanation
+
+Binary search on the answer for the minimum number of tickets, which we call $t$, to sell to make the total ecological contribution to at least $k$. 
+
+Greedily, it is optimal to maximize the total revenue from selling these tickets since that will minimize the total number of tickets. If it is impossible to sell at least $k$ tickets, then output $-1$.
```

Comment:
typo still not fixed. Should be $k$ revenue not tickets.

Also, 1st sentence may be better phrased as "Let's try to determine the maximum revenue ....."